gatsby-config.js

```diff
@@ -139,5 +139,53 @@ module.exports = {
         },
       },
     },
+    {
+      resolve: 'gatsby-plugin-local-search',
+      options: {
+        name: 'learn-pages',
+        engine: 'flexsearch',
+        engineOptions: 'speed',
```

Comment:
`engineOptions: { tokenize: "forward" }` seems to give much better search results as it does fuzzy matching. Not sure what the performance difference (if any) is though 🤷

Reply:
I just tried this. I agree it's way faster.